reference_guide/api_changes_list_2024.md

```diff
@@ -147,6 +147,14 @@ Class `com.intellij.diff.editor.DiffVirtualFile` now extends `com.intellij.diff.
 `com.intellij.openapi.util.io.NioPathUtil.isAncestor(Path, Path, boolean)` method removed
 : Use `Path.startsWith()` instead.
 
+`com.intellij.util.CachedValueBase.setData(CachedValueBase.Data)` abstract method added
+: Must be implemented
+
+`com.intellij.util.CachedValueBase.getRawData()` abstract method added
+: Must be implemented
```

Comment:
description should end with `.`

Reply:
Would be great to have some tooling to detect that in IDE